src/content/docs/apis/nerdgraph/examples/topology-nerdgraph-tutorial.mdx

```diff
@@ -0,0 +1,204 @@
+
+## Requirements [#requirements]
+
+Topology correlation is currently in limited release. See [topology requirements](/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions/#topology-requirements).
+
+To use NerdGraph, you'll need a [user key](/docs/apis/nerdgraph/get-started/introduction-new-relic-nerdgraph/#authentication). 
+
+## Overview of tutorial [#overview]
+
+With New Relic’s [Applied Intelligence](/docs/alerts-applied-intelligence/new-relic-alerts/get-started/introduction-applied-intelligence/, you can create custom [decisions](/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions) that govern how your incidents are correlated. One kind of custom decision logic uses the concept of “topology,” which in this context is a representation of your service map. 
+This doc will explain how to use our [NerdGraph API](/docs/apis/nerdgraph/get-started/introduction-new-relic-nerdgraph/) to configure your topology mapping. 
+
+This tutorial will give example NerdGraph calls that: 
+* Set up your topology by creating vertices and edges
+* Delete vertices and edges
+* Retrieve topology data
+
+Before using NerdGraph to configure your topology, you should have a basic understanding of: 
+
+* [What Applied Intelligence decisions are](/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions) and [how topology correlation works](/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions/#topology).
+* The topology structure you're trying to implement. One way to understand this is to use our [service map feature](/docs/understand-dependencies/understand-system-dependencies/service-maps/introduction-service-maps/) to see how entities in your infrastructure relate to each other.  
+* [What NerdGraph is](/docs/apis/nerdgraph/get-started/introduction-new-relic-nerdgraph) and [how to use the NerdGraph API explorer](/docs/apis/nerdgraph/get-started/introduction-new-relic-nerdgraph/#explorer) to run queries.
+
+## Mutation examples [#mutations]
+
+In NerdGraph, mutations are requests that perform an action ([learn more about NerdGraph terminology](/docs/apis/nerdgraph/get-started/introduction-new-relic-nerdgraph/#terminology)). 
+
+In this section, we’ll show you how to use the `aiTopologyCollector` query to create, edit, or delete your topology. In this example, we’ll set up the following service map:
+
+![Topology example](./images/topology-example.png "topology example")
```

Comment:
I like this chart. I'm curious why there are arrows between the services but not the services and hosts. Maybe I'm reading too much into it :)

Reply:
i don't know why they chose these images.

Reply:
You can just this one from the other doc: <figcaption>In this service map, the hosts and apps are the vertices, and the lines showing their relationships are the edges.</figcaption>